README.md

```diff
@@ -34,6 +34,17 @@ Code style follows prettier conventions (`yarn prettier`). Commit messages follo
   - `REACT_APP_INFURA_PROJECT_ID` should remain blank because we are currently using Alchemy as our provider
   - `REACT_APP_ALCHEMY_API_KEY` alchemy api key should be provided by the contributor if he hits api limits
 
+### @yearn/web-lib access
+
+You will need to generate a [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), and create or update the ~/.npmrc file with the following content:
+
+```
+always-auth=true
+@yearn:registry=https://npm.pkg.github.com/
+registry=https://registry.npmjs.org
+//npm.pkg.github.com/:_authToken=YOUR_ACCESS_TOKEN
```

Comment:
```suggestion
//npm.pkg.github.com/:_authToken=YOUR_ACCESS_TOKEN
```
Since we are updating the global `.npmrc` file located in home, and not the one inside the project, we only need the suggested line